tools/docker/tests/test_flows.sh

```diff
@@ -180,6 +180,48 @@ test_client_steering_mandate() {
         'grep -i -q "ACK_MESSAGE" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
 }
 
+test_client_association_dummy(){
+    status "test client association dummy"
+    check_error=0
+
+    dbg "Connect dummy STA to wlan0"
+    check docker exec -it repeater1 sh -c \
+        'echo "AP-STA-CONNECTED 11:22:33:44:55:66" > /tmp/$USER/beerocks/wlan0/EVENT'
+    
+    dbg "Send client association control request to the chosen BSSID to steer the client (UNBLOCK) "
+    eval send_bml_command "client_allow \"11:22:33:44:55:66 aa:bb:cc:00:00:20\"" $redirect
+    sleep 1
+
+    dbg "Confirming Client Association Control Request message was received (UNBLOCK)"
+    check docker exec -it repeater1 sh -c \
+        'grep -i -q "Got client allow request" /tmp/$USER/beerocks/logs/beerocks_agent_wlan2.log'
+
+    dbg "Send client association control request to all other (BLOCK) "
+    eval send_bml_command "client_disallow \"11:22:33:44:55:66 aa:bb:cc:00:00:10\"" $redirect
```

Comment:
I would prefer to use `$mac_agent1_wlan0` here instead of an explicit MAC address. However, at the moment `$mac_agent1_wlan0` is in hex notation instead of MAC notation. But I think that since commit 1cd4619d62c44 it is no longer needed to use the hex notation. If that is true, I would like a first commit that removes the mac_to_hex for the radio addresses, and then use `$mac_agent1_wlan0` here.

Reply:
I agree with you. Do you think we need to remove the mac_to_hex method entirely ? from functions.sh also? @arnout

Reply:
> Do you think we need to remove the mac_to_hex method entirely ?

If it is no longer used, yes we can remove it. However, it might still be needed in some other context (i.e. outside of a tlv).

Reply:
> > Do you think we need to remove the mac_to_hex method entirely ?
> 
> If it is no longer used, yes we can remove it. However, it might still be needed in some other context (i.e. outside of a tlv).

It is still used in `test_higher_layer_data_payload_trigger` and I think it's appropriate to keep that use because the UCC generates it like that (i.e. hex notation, not colon notation). So no, don't remove it from `functions.sh`.